core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1037,6 +1075,34 @@ private void sendNotificationBroadcast(int type, int code) {
         sendBroadcast(intent);
     }
 
+    private void skipIfMetEnding() {
```

Comment:
`positionSaverTick` is only executed every 5 seconds. I think you can use EventBus:

```
@Subscribe
public void skipIfMetEnding(PlaybackPositionEvent event) {
```

Reply:
instead of a new Subscriber, I'm piggybacking on the onPositionObserver()